### ISSUE TITLE
Update Rows with and keep sorting active

### ISSUE THREAD

**Issue by Danielku15** [mleibman/SlickGrid#938]
I am using SlickGrid to display real time data I get from a WebSocket. I use updateItem on the DataView to update the single rows according to the latest data. This works fine so far. The problem starts when you allow sorting of the items because calling updateItem does not ensure the row gets moved to the correct place. 

An example usage could be a Web Task Manager. You sort by CPU load and of course if the CPU load changes, the high load Processes should move to top. 

Of course I could simply call sort on the whole DataView but that would consume unnessecary processing power since the whole data is already sorted, only one item is at the wrong place. Is there a built-in way to tell SlickGrid that only this row is at the wrong place so it uses the current sorting to place it correctly?

I thought of creating my own DataView where I implement the updateItem in a way that does a remove+sorted insert but I'm not yet sure if this would make SlickGrid render correctly.

**Comment by JohnRanger** [comment]
@Danielku15 ,
have you ever managed to optimize slickgrid (your personal copy of it) in the optimized way you have described it above? (As I was looking into this myself but did not succeed).

With kind regards,

John

**Comment by 6pac** [comment]
strangely, we are just discussing this over at the 6pac repo:   https://github.com/6pac/SlickGrid/issues/353

you should be using that repo - this one is dead - see slickgrid.net

**Comment by Danielku15** [comment]
@JohnRanger No, I did not make any SlickGrid improvements as part of my thesis or our product. Since this issue was opened we are having a separate timer for triggering a sort. We are in the lucky situation that even though we have a lot of table rows (>1300 nowadays) with updates per row every ~5 seconds, our values do not make such huge jumps that it would require fully dynamic reordering.
